tasks/delete_app_objects_kind.yml

```diff
@@ -29,6 +29,6 @@
       - app={{ app.name }}
       - deployment_stamp={{ targeted_deployment_stamp }}
   register: remaining_objects
-  until: ( remaining_objects.resources | length ) == 0
+  until: remaining_objects.failed == True or ( remaining_objects.resources | length ) == 0
```

Comment:
I am pretty sure this works, but I cannot find in Ansible's documentation a reference to this `failed` attribute for registered variables. Maybe this is not documented and you find it out by inspecting this variable upon failure... :man_shrugging:

Reply:
I found this in Ansible's documentation about [common return values](https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#failed).

Reply:
![](https://media0.giphy.com/media/NEvPzZ8bd1V4Y/giphy.gif)